packages/rum-core/test/emulate/mockPerformanceObserver.ts

```diff
@@ -13,7 +13,7 @@ export function mockPerformanceObserver({ typeSupported = true, emulateAllEntryT
   let performanceObserver: PerformanceObserver
   let bufferedEntries: RumPerformanceEntry[] = []
 
-  const mock = (callback: PerformanceObserverCallback) => {
+  const mock = function (callback: PerformanceObserverCallback) {
```

Comment:
PerformanceObserver need to be a constructor. This used to work because arrow function were transpiled to function (es5 target)

Reply:
Interesting. Could we make it a `class` instead?

Reply:
yes, done ✅ 
It's a bit more readable!